pyproject.toml

```diff
@@ -49,10 +49,11 @@ psycopg2-binary = {version = "^2.9.9", optional = true }
 psycopg2 = {version = "^2.9.9", optional = true }
 alembic = {version = "^1.13.3", optional = true }
 matplotlib = {version = "^3.8.4", optional = true }
+backoff = {version = "2.2.1", optional = true }
```

Comment:
This is may become a problem for us. The library is no longer maintained.[1] While it should have everything we need at this point, we need to be aware of the fact.

[1] https://github.com/litl/backoff/issues/211

Reply:
we discussed at the standup that pinning down the version should prevent us from future problems.